rust-sdk/core/src/math/tick_array.rs

```diff
@@ -258,9 +257,8 @@ mod tests {
     #[test]
     fn test_get_next_initializable_tick_out_of_bounds() {
         let sequence = test_sequence(16);
-        let pair = sequence.next_initialized_tick(2817);
```

Comment:
```
test sequence: [-1408      0) [0         1408) [1408          2816)
               start = -1408                                    end=2815
```
Then
- `sequence.next_initialized_tick(2813) -> Ok((None, 2815))`
- `sequence.next_initialized_tick(2814) -> Ok((None, 2815))`
- `sequence.next_initialized_tick(2815) -> Err(INVALID_TICK_ARRAY_SEQUENCE )` # next is NOT inclusive
- `sequence.next_initialized_tick(2816) -> Err(INVALID_TICK_ARRAY_SEQUENCE )`
- `sequence.next_initialized_tick(2817) -> Err(INVALID_TICK_ARRAY_SEQUENCE )`